From 5e830c7c547daf781a93b69140bdded5609b6f08 Mon Sep 17 00:00:00 2001
Message-Id: <5e830c7c547daf781a93b69140bdded5609b6f08.1561706494.git.aditya@kobol.io>
From: Aditya Prayoga <aditya@kobol.io>
Date: Fri, 28 Jun 2019 15:21:18 +0800
Subject: [PATCH] arm: dts: armada-38x: Fix broken SPI support after dts sync

SPI support on mainline u-boot is broken since
commit 30c4383 (ARM: mvebu: sync Armada-38x dts with Linux 4.20)

revert the changes on SPI node only.

Signed-off-by: Aditya Prayoga <aditya@kobol.io>
---
 arch/arm/dts/armada-38x.dtsi | 48 ++++++++++++++++++------------------
 1 file changed, 24 insertions(+), 24 deletions(-)

diff --git a/arch/arm/dts/armada-38x.dtsi b/arch/arm/dts/armada-38x.dtsi
index 72c49beb71..f21a817444 100644
--- a/arch/arm/dts/armada-38x.dtsi
+++ b/arch/arm/dts/armada-38x.dtsi
@@ -133,6 +133,30 @@
 				clocks = <&coreclk 2>;
 			};
 
+			spi0: spi@10600 {
+				compatible = "marvell,armada-380-spi",
+						"marvell,orion-spi";
+				reg = <0x10600 0x50>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				cell-index = <0>;
+				interrupts = <GIC_SPI 1 IRQ_TYPE_LEVEL_HIGH>;
+				clocks = <&coreclk 0>;
+				status = "disabled";
+			};
+
+			spi1: spi@10680 {
+				compatible = "marvell,armada-380-spi",
+						"marvell,orion-spi";
+				reg = <0x10680 0x50>;
+				#address-cells = <1>;
+				#size-cells = <0>;
+				cell-index = <1>;
+				interrupts = <GIC_SPI 63 IRQ_TYPE_LEVEL_HIGH>;
+				clocks = <&coreclk 0>;
+				status = "disabled";
+			};
+
 			gic: interrupt-controller@d000 {
 				compatible = "arm,cortex-a9-gic";
 				#interrupt-cells = <3>;
@@ -620,30 +644,6 @@
 			no-memory-wc;
 			status = "disabled";
 		};
-
-		spi0: spi@10600 {
-			compatible = "marvell,armada-380-spi",
-					"marvell,orion-spi";
-			reg = <MBUS_ID(0xf0, 0x01) 0x10600 0x50>;
-			#address-cells = <1>;
-			#size-cells = <0>;
-			cell-index = <0>;
-			interrupts = <GIC_SPI 1 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&coreclk 0>;
-			status = "disabled";
-		};
-
-		spi1: spi@10680 {
-			compatible = "marvell,armada-380-spi",
-					"marvell,orion-spi";
-			reg = <MBUS_ID(0xf0, 0x01) 0x10680 0x50>;
-			#address-cells = <1>;
-			#size-cells = <0>;
-			cell-index = <1>;
-			interrupts = <GIC_SPI 63 IRQ_TYPE_LEVEL_HIGH>;
-			clocks = <&coreclk 0>;
-			status = "disabled";
-		};
 	};
 
 	clocks {
-- 
2.17.1

